.pipelines/templates/aks-setup.yaml

```diff
@@ -87,6 +77,15 @@ steps:
         az aks nodepool add -g ${AZURE_CLUSTER_NAME} --cluster-name ${AZURE_CLUSTER_NAME} --os-type Windows --name win --node-count 1 ${EXTRA_ARGS:-} > /dev/null
       fi
 
+      # add gpu node pool
+      if [[ ${{ parameters.testWithGPU }} == True ]]; then
+        echo "adding gpu node pool"
+        az extension add --name aks-preview
+        az extension update --name aks-preview
+
```

Comment:
I see we're no longer setting the `AZURE_LOCATION` to `eastus2euap`. Just to confirm this is only available in westus other than eastus2euap?